widgetssdk/src/main/java/com/glia/widgets/core/callvisualizer/domain/CallVisualizer.java

```diff
@@ -21,6 +22,7 @@ public interface CallVisualizer {
      * For each subsequent call thereafter, the same code will be returned as long as the code has not
      * expired. During that time, the code can be used to initiate an engagement. Once the operator
      * has used the visitor code to initiate an engagement, the code will expire immediately.
+     * TODO ADD CONFIGURATION OR REFACTOR TO INIT
```

Comment:
Not sure I understand this TODO, is it regarding runtime theme configurations?

Reply:
Leftover todo from when init had not yet been changed